docs/projects/ansible/collection/centurion/playbooks/teams.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Teams
+description: Creation and patching of teams and permissions.
```

Comment:
product name is required. this is what search engines uses as part of the page description.

Reply:
fixed